NodeJS/CLI Password Generator/README.md

```diff
@@ -0,0 +1,51 @@
+# CLI Password Generator
+
+## Description 
+Command line app to generate random passwords. This uses crypto module due to which generated passwords are totally random, and unpredictable.
+
+## Tech Stack
+
+NodeJS 
+
+## Demonstration
+
+![Password Generator](utils/passgen.png)
+
+## Usage
+
+Install dependencies
+
+```
+npm install
+```
```

Comment:
also add npm and node js from the website and mention website for downloading node.js